frontstage/templates/passwords/forgot-password.check-email.html

```diff
@@ -5,10 +5,9 @@
 
 {% block main %}
 
-    <h1 class="saturn">Check your email</h1>
+    <h1 class="saturn">Password reset request sent</h1>
 
-    <p>If your account is verified, you will receive an email at the registered email address.</p>
-    <p>Please follow the link in the email to reset your password.</p>
+    <p>If {{ email_address }} is registered to an account, you'll receive an email with instructions to reset your password</p>
```

Comment:
This needs to be `email` as thats what you've called it when rendering this page, or you can change the kwarg when its rendered

Reply:
I also think calling it `email_address` is better than just `email` small thing though